alns/stopping_criteria/MaxRuntime.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+from alns.State import State
+from alns.stopping_criteria.StoppingCriterion import StoppingCriterion
+
+
+class MaxRuntime(StoppingCriterion):
+    def __init__(self, max_runtime: int) -> None:
```

Comment:
Remark: if the function has not explicit return value (that is, there is no `return` statement), I do not expect a return type. There's no need to annotate `-> None` here and in other initialisers.